robottelo.properties.sample

```diff
@@ -154,7 +154,10 @@ ssh_key=
 
 # Fake manifest testing.
 # [fake_manifest]
-# URL of the base manifest
+# Comma-separated list of urls for multiple manifessts
+# e.g. url=name1=http://url/1,name2=http://url/2,...
+# url=http://url is supported for backward compatibility and is equal to:
+# url=default=http://url
 # url=http://example.org/valid-redhat-manifest.zip
```

Comment:
this line became redundant. Btw, since 3 files with different extensions are set in this section, i'd prefer including extension (`.zip`) in filename examples.